test/config/loader.js

```diff
@@ -1,9 +1,12 @@
-const path = require('path');
-const test = require('@ava/test');
-const {loadConfig} = require('../../lib/load-config');
+import path from 'path';
+import {fileURLToPath} from 'url';
+
+import test from '@ava/test';
+
+import {loadConfig} from '../../lib/load-config.js';
 
 const CWD = process.cwd();
-const FIXTURE_ROOT = path.resolve(__dirname, '../../test-tap/fixture/load-config');
+const FIXTURE_ROOT = path.resolve(path.dirname(fileURLToPath(import.meta.url)), '../../test-tap/fixture/load-config');
```

Comment:
```suggestion
const FIXTURE_ROOT = fileURLToPath(new URL('../../test-tap/fixture/load-config', import.meta.url));
```

Could be slightly simplified.